### PR TITLE
3.2 prereqs

### PR DESCRIPTION
**Pull request by isantop**
Merge in syntactic, formatting, and documentation changes from Kernelstub 3.2.

Required by #33

### CODE REVIEW

**Comment by jackpot51** [comment]
The changes appear to be ok to me

**Comment by jackpot51** [comment]
My current entries now point to the previous paths with this installed

**Comment by jackpot51** [comment]
IMG_20190716_152226

**Comment by isantop** [comment]
@jackpot51 I just noticed that as well. Should be fixed as of cf159cf

File: kernelstub/application.py [hunk 1]
```diff
@@ -39,50 +39,43 @@
...
+SYSTEMD_SUPPORT = False
+try:
+    from systemd.journal import JournalHandler
+    SYSTEMD_SUPPORT = True
+
+except ImportError:
     pass
 
+class CmdLineError(Exception):
+    """ Exception raised when we can't find any kernel parameters """
```
Comment (crawfxrd) [inline kernelstub/application.py]: Is `pass` not needed?

File: kernelstub/opsys.py [hunk 2]
```diff
@@ -122,11 +134,12 @@ def strip_quotes(self, value):
...
+        """Return a list with the current OS release data."""
         try:
             with open('/etc/os-release') as os_release_file:
                 os_release = os_release_file.readlines()
         except FileNotFoundError:
-            os_release = ['NAME="%s"\n' % self.name,
+            os_release = ['NAME="{}"\n'.format(self.name),
                           'ID=linux\n',
                           'ID_LIKE=linux\n',
                           'VERSION_ID="%s"\n' % self.version]
```
Comment (crawfxrd) [inline kernelstub/opsys.py]: `format()` here for consistency?

**Pull request closed without merging**